OWNERS

```diff
@@ -1,18 +1,14 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - anishasthana
-  - hemajv
   - HumairAK
   - tumido
-  - martinpovolny
-  - 4n4nd
-  - durandom
 
 reviewers:
   - anishasthana
   - hemajv
-  - HumairAK
```

Comment:
@tumido Weren't we planning on keeping a small overlap between approvers and reviewers?

Reply:
Ah, you're right. :+1:  Especially with the https://github.com/thoth-station/thoth-application/pull/805 settings change, it make sense. @goern can you please change this back?

Reply:
well, not being in the reviewers list just prevents someone from being picked and assigned as a reviewer. see https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process 

or do you want Humair to be a reviewer so he can add the lgtm label?!

Reply:
I think we both want to be reviewers as well. I mean, even if I'm an approver I might want to `lgtm` something while still wanting a second pair of eyes on that PR before merging.. Do I make sense or is that totally wrong expectation I have?